src/main/java/org/influxdb/InfluxDBException.java

```diff
@@ -126,7 +126,7 @@ public boolean isRetryWorth() {
     }
   }
 
-  private static InfluxDBException buildExceptionFromErrorMessage(final String errorMessage) {
+  public static InfluxDBException buildExceptionFromErrorMessage(final String errorMessage) {
```

Comment:
Why public, if you want to test this like below, write the test in the matching package.

Reply:
Please add a changelog.md entry with pointers where the usage of influxdb-java in a reactive environment is shown.

Reply:
Because I want to reuse this method in influxdb-java-reactive to parse errors from HTTP header X-Influx-Error. If its is a problem I can use something like this:

```java
String errorBody = String.format("{\"error\":\"%s\"}", errorHeader);
```

Can I let this method public?

Reply:
The CHANGELOG.md was updated.